components/builder-web/app/dashboard/_dashboard.component.scss

```diff
@@ -0,0 +1,275 @@
+.hab-dashboard {
+
+  .page-title {
+
+    h3.username {
+      font-family: $base-font-family;
+      font-weight: normal;
+    }
+  }
+
+  section {
+    @include row;
+
+    &.intro {
+      background-color: $hab-blue;
+      border-radius: 5px;
+      height: 84px;
+      margin-bottom: 44px;
+
+      .image, .text, .buttons {
+        height: 100%;
+      }
+
+      .image {
+        @include span-columns(2 of 12, 'block-collapse');
+        position: relative;
+        text-align: center;
+
+        img {
+          position: relative;
+          height: 96px;
+          top: 2px;
+        }
+      }
+
+      .text, .buttons {
+        position: relative;
+
+        > div {
+          position: absolute;
+          top: 50%;
+          transform: translateY(-50%);
+        }
+      }
+
+      .text {
+        @include span-columns(6 of 12);
+        font-size: rem(12);
+        color: $hab-white;
+
+        strong {
+          font-size: rem(14);
+          display: block;
+          margin-bottom: 4px;
+        }
+
+        p {
+          margin: 0;
+          line-height: rem(18);
+        }
+      }
+
+      .buttons {
+        @include span-columns(4 of 12);
+        text-align: right;
+
+        a {
+          font-size: rem(12);
+          padding: 5px 15px;
+          margin: 0 10px;
+        }
+      }
+    }
+
+    &.upper {
+      height: 230px;
+      border-radius: 5px;
+      background-color: $hab-white;
+      box-shadow: 0 1px 25px 0 $very-light-gray;
+      margin-bottom: 30px;
+
+      .origins, .top, .favorites {
+        @include span-columns(4, 'block-collapse');
+        height: 100%;
+      }
+
+      .top, .favorites {
+        border-left: 1px solid $very-light-gray;
+      }
+
+      .origins .heading {
+        border-radius: 5px 0 0 0;
+      }
+
+      .favorites .heading {
+        border-radius: 0 5px 0 0;
+      }
+
+      .heading {
+        position: relative;
+        padding: 10px 20px;
+        background-color: $very-light-gray;
+
+        h4 {
+          margin: 0;
+          font-family: $base-font-family;
+          text-transform: none;
+          font-size: rem(16);
+          line-height: rem(24);
+        }
+
+        .controls {
+          position: absolute;
+          right: 20px;
+          top: 50%;
+          transform: translateY(-50%);
+
+          a {
+            display: inline-block;
+            margin-left: rem(16);
+          }
+        }
+      }
+
+      .content {
+        ul {
+          padding: 12px 0;
+
+          li {
+            position: relative;
+
+            &:hover {
+              font-weight: bold;
+              background-color: rgba(232,240,246,0.2);
```

Comment:
Probably a good idea, yeah. Good spot! I'll check these against existing ones and make new ones where it makes sense.